src/components/shared/TokenInfoPopover/TokenInfoPopover.tsx

```diff
@@ -1,31 +1,31 @@
 import React from 'react';
 import { isEmpty } from 'lodash';
 
-import { AnyToken } from '~data/index';
+import { Token } from '~types';
 
 import TokenInfo from './TokenInfo';
-import NotAvailableMessage from './NotAvailableMessage';
+import NotAvailableMessage from '../NotAvailableMessage/NotAvailableMessage';
 
-import styles from './InfoPopover.css';
+import styles from './TokenInfoPopover.css';
 
 interface Props {
-  token?: AnyToken;
+  token?: Token;
   isTokenNative: boolean;
 }
 
-const displayName = 'InfoPopover.TokenInfoPopover';
+const displayName = 'TokenInfoPopover';
 
 const TokenInfoPopover = ({ token, isTokenNative }: Props) => {
   return (
-    <>
+    <span>
```

Comment:
Why was this changed to a `span` element ?

Reply:
What I have noticed is that there are many places in the code that use a jsx conditional statement and are surrounded by either fragments, span or div because these conditional statements cause errors when they are not enclosed in a parent element. When it's a fragment, it gives the "Fragments should contain more than one child" error. I have not found a solution for this yet so I thought a span could work for now. 😅  but I am open to suggestions.

Reply:
@rdig just wanted to follow up on this specifically as I keep encountering it and I would like to hear your opinion on the best way to solve this.

Reply:
@Julianahlk I also came across this error and having looked into this, I think it might be best to modify the eslint config for the rule triggering it (as I did in my [extensions PR](https://github.com/JoinColony/colonyCDapp/pull/95/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5))

Reply:
Thanks! This works 👍